### PR TITLE
feat: add extra volume and volumemount to helm template

### PR DESCRIPTION
**Pull request by sp3nx0r**
We ran into the use case where we're using Vault CSI and secrets won't get created without a volume to initiate the process. So submitting this PR to add the ability to add extra volumes as necessary.

### CODE REVIEW

File: helm-charts/whitesource-renovate/templates/deployment.yaml [hunk 1]
```diff
@@ -133,6 +133,9 @@ spec:
             {{- end }}
             - name: {{ .Release.Name }}-cache-volume
               mountPath: /tmp/renovate
+          {{- if ne (len .Values.extraVolumeMounts) 0 }}
+{{ toYaml .Values.extraVolumeMounts | indent 12 }}
```
Comment (rarkins) [inline helm-charts/whitesource-renovate/templates/deployment.yaml:137]: This leading whitespace seems wrong, or is it intentional?

Comment (sp3nx0r) [reply helm-charts/whitesource-renovate/templates/deployment.yaml]: It's intentional for `indent` but swapped it out for `nindent | trim` which appears to be the preferred way to do this.

**Comment by sp3nx0r** [comment]
testing for this can be done by uncommenting the attributes in `values.yaml` and `helm template`

```
extraVolumes:
  - name: secrets-store-inline
    csi:
      driver: secrets-store.csi.k8s.io
      readOnly: true
      volumeAttributes:
        secretProviderClass: "some-secret-store"

extraVolumeMounts:
  - name: secrets-store-inline
    mountPath: "/mnt/secrets-store"
    readOnly: true
```

**Comment by sp3nx0r** [comment]
@rarkins any hope on getting this merged?

**Review by rarkins — APPROVED** [review]

**Pull request merged**